boards/arm/efr32_radio/doc/index.rst

```diff
@@ -9,16 +9,19 @@ EFR32 Radio Boards
    brd4104a.rst
    brd4250b.rst
    brd4180a.rst
+   brd4255a.rst
 
 Overview
 ********
 
-Support for EFR32 Radio boards is provided by one of the starter kits
+Support for EFR32 Radio boards is provided by one of the starter kits or radio boards
 
 - `SLWSTK6020B Bluetooth SoC Starter Kit`_
 - `SLWSTK6000B Mighty Gecko Wireless Starter Kit`_
 - `SLWSTK6061B Proprietary Wireless Starter Kit`_
 - `SLWSTK6006A Mighty Gecko Wireless Starter Kit`_
+- `SLWRB4255A Flex Gecko Wireless Radio Board`_
```

Comment:
It looks like SiLabs does not provide dedicated starter kit for the SLWRB4255A / BRD4255A radio board. One needs to buy a starter kit for one of the other radio boards or [Wireless Starter Kit Mainboard](https://www.silabs.com/development-tools/wireless/wireless-starter-kit-mainboard).

In any case, this entry is incorrect. We should either remove it or replace by 
- SLWMB4001A Wireless Starter Kit Mainboard

Reply:
Thanks @mnkp , I added " or radio boards" to line 17 to denote that not all the boards on that list come as part of a starter kit, but maybe it should have been clearer. Perhaps, I should keep the original starter kits list, and add another list of supported radio boards, mentioning that the main board should be purchased separately?

Reply:
This list was only meant for starter kits. The new board is already listed under "EFR32 Radio Boards" on line 12.